### PR TITLE
pliss blog post

### PR DESCRIPTION
**Pull request by reallyTG**
blog post for pliss 2019

### CODE REVIEW

File: blog/_src/posts/2019-03-09-pliss-learn-about-pl-implementation-in-a-castle.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+from both academia and industry, each well-versed in the practical side of
+programming languages.
+The lectures cover current research as well as future trends in programming language
+design and implementation, including:
+
+- Developing Security-Aware Languages with Cristina Cifuentes;
+- Semantics-First Language Design with Sylvan Clebsch;
+- Compiler Design Patterns for Machine Learning by Albert Cohen;
+- A Survey of V8 and WebAssembly by Ben L. Titzer;
+- Crafting User-Friendly Compilers by Nicholas Batsakis;
```
Comment (howell) [inline blog/_src/posts/2019-03-09-pliss-learn-about-pl-implementation-in-a-castle.md]: Batsakis => Matsakis

**Comment by howell** [comment]
Looks good to me.

**Comment by bennn** [comment]
thank you @reallyTG

**Comment by reallyTG** [comment]
😎

**Pull request merged**